TestDockerfile37

```diff
@@ -1,14 +1,29 @@
 FROM python:3.7-slim
 WORKDIR /app
-RUN apt-get update -y && apt-get install -y python3-dev libcurl4-openssl-dev libgnutls28-dev libjpeg-progs libimage-exiftool-perl gifsicle scons python3-all-dev libboost-python-dev libexiv2-dev ffmpeg make zlib1g-dev gcc libssl-dev libjpeg-dev libwebp-dev redis && apt-get clean
-RUN apt-get update -y && apt-get install --reinstall -y build-essential && apt-get clean
-RUN pip install --upgrade pip
-COPY setup.py /app/setup.py
-RUN mkdir -p /app/thumbor
-RUN touch /app/thumbor/__init__.py
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends --no-install-suggests \
+        python3-dev \
+        libcurl4-openssl-dev \
+        libgnutls28-dev \
+        libjpeg-progs \
+        libimage-exiftool-perl \
+        gifsicle \
+        scons \
+        python3-all-dev \
+        libboost-python-dev \
+        libexiv2-dev \
+        ffmpeg \
+        make \
+        zlib1g-dev \
+        gcc \
+        libssl-dev \
+        libjpeg-dev \
+        libwebp-dev \
+        redis && \
+    apt-get install --reinstall --no-install-recommends --no-install-suggests -y build-essential && \
```

Comment:
Seems to me like this line could use a `-y`
```suggestion
    apt-get install --reinstall -y --no-install-recommends --no-install-suggests -y build-essential && \
```

Reply:
But it has already, it's before the first package

Reply:
True. I clearly missed that (along with the fact that it builds lol). That may be an indicaiton taht we're better keep those lines consistent throughout those Dockerfiles. I'll edit the suggestions.